src/couch_srt/README.md

```diff
@@ -206,20 +207,20 @@ The query and HTTP API's are well documented and tested (h/t @iilyak) and
 provide an excellent overview of the interaction patterns and query capabilities
 of CSRT. Those can be found at:
 
-* `csrt_query.erl` "Query API functions"
-  - https://github.com/apache/couchdb/blob/93bc894380056ccca1f77415454e991c4d914249/src/couch_stats/src/csrt_query.erl#L319-L674
+* `couch_srt_query.erl` "Query API functions"
+  - https://github.com/apache/couchdb/blob/93bc894380056ccca1f77415454e991c4d914249/src/couch_stats/src/couch_srt_query.erl#L319-L674
```

Comment:
The sha of this link is most likely need to be updated

Reply:
I did update all links

Reply:
Thank you!